docs/source/user_guide/config.rst

```diff
@@ -0,0 +1,50 @@
+.. include:: ./../ext_links.txt
+
+Configuration Options for ``numba-dpex``
+========================================
+
+``numba-dpex`` provides a set of environment variables and flags for configuring different aspects of the compilation, debugging and execution of programs. The configuration flags of ``numba-dpex`` are mostly inherited from those of Numba*. They are defined in :file:`numba_dpex/core/config.py` and are listed as follows:
+
+``SAVE_IR_FILES``:
```

Comment:
It should be mentioned that when setting these configuration variables using environment variables, the name of the environment variable is `NUMBA_DPEX_<name of config>`

Reply:
I explained it on line 41.

Reply:
Can we move the note on top of the list? That way no one should miss it.

Reply:
> Can we move the note on top of the list? That way no one should miss it.

Fixed.